fmriprep/workflows/bold/base.py

```diff
@@ -516,18 +528,22 @@ def init_func_preproc_wf(bold_file):
             # update name source for optimal combination
             (inputnode, func_derivatives_wf, [
                 (('bold_file', combine_meepi_source), 'inputnode.source_file')]),
+            (bold_bold_trans_wf, join_echos, [
+                ('outputnode.bold', 'bold_files')]),
+            (join_echos, final_boldref_wf, [
+                ('bold_files', 'inputnode.bold_file')]),
```

Comment:
Just taking the first echo, right?

```suggestion
                (('bold_files', pop_file), 'inputnode.bold_file')]),
```

Reply:
The recent changes to the reference workflow (https://github.com/nipreps/niworkflows/pull/408) should support multiple files. Inside the workflow it should currently grab the first echo.

Reply:
Okay, try 270fdd4dc5467aff448ca86f62b7c0393c76f551.